### DISCUSSION TITLE
advanced rotation

### DISCUSSION THREAD

**Discussion by stormneed** [fribbels/hsr-optimizer#787] [Q&A]
I see an advanced_rotation.md in the git, however I can't seem to find the page for it in the site, I might be blind so can someone guide me pls ^^'

**Comment by FI00ds** [comment]
When you're on the optimized tab, on the right side of the second row you can customise your combo rotation.
If you toggle advanced mode you'll see the advanced combo.
The guide isn't in the website itself, we just link to the one you saw.
